docs/source/deploying-volttron/proxy_web_server.rst

```diff
@@ -0,0 +1,277 @@
+cert and key file paths. These can be generated using volttron through vcfg, or can be generated by a third-party, i.e.
+`Let's Encrypt <https://letsencrypt.org/>`_. Once that is complete, verify that ProxyPass and ProxyPassReverse options,
+as well as the /vc path found under the RewriteRule option are all appropriate for your use case. This simple use-case
+routes all web traffic through the proxy to your VOLTTRON instance running on your localhost.
+
+
+.. code-block:: console
+
+    <IfModule mod_ssl.c>
+        SSLPassPhraseDialog  exec:/usr/share/apache2/ask-for-passphrase
+        SSLSessionCache		shmcb:${APACHE_RUN_DIR}/ssl_scache(512000)
+        SSLSessionCacheTimeout  300
+        SSLRandomSeed startup file:/dev/urandom  256
+        SSLRandomSeed connect builtin
+        SSLCryptoDevice builtin
+
+        <VirtualHost _default_:443>
+            ErrorLog logs/ssl_error_log
+            TransferLog logs/ssl_access_log
+            LogLevel warn
+            SSLEngine on
+            # Require TLS 1.2 only
+            SSLProtocol -all +TLSv1.2
+            # Require Strong Ciphers
+            SSLCipherSuite "EECDH:!RC4:!3des:!SHA"
+            # Require Cipher Order
+            SSLHonorCipherOrder on
+            SSLCertificateFile <path to cert file provided by CA>
+            SSLCertificateKeyFile <path to key file>
+            <Files ~ "\.(cgi|shtml|phtml|php3?)$">
+                SSLOptions +StdEnvVars
+            </Files>
+            BrowserMatch "MSIE [2-5]" \
+                     nokeepalive ssl-unclean-shutdown \
+                     downgrade-1.0 force-response-1.0
+            CustomLog logs/ssl_request_log \
+                      "%t %h %{SSL_PROTOCOL}x %{SSL_CIPHER}x \"%r\" %b"
+
+            #Setup proxy for volttron
+            ProxyRequests Off
+            ProxyPreserveHost Off
+            ProxyVia Off
+            # Set proxy path as appropriate.
+            ProxyPass          /     http://localhost:8080/ timeout=60
+            ProxyPassReverse   /     http://localhost:8080/ timeout=60
+
+            #Setup websockets to proxy for volttron central
+            RewriteEngine On
+            RewriteCond %{HTTP:UPGRADE} ^WebSocket$ [NC]
+            RewriteCond %{HTTP:CONNECTION} Upgrade$ [NC]
+            # Set proxy path as appropriate
+            RewriteRule /vc/index.html#/dashboard(.*) ws://localhost:8080/vc/index.html#/dashboard$1 [P]
+
+            #Add HSTS header:
+            Header always set Strict-Transport-Security "max-age=31536000"
+
+        </VirtualHost>
+    </IfModule>
+
+Setting Up Apache2
+^^^^^^^^^^^^^^^^^^
+
+Once all the config files have been properly configured, use them to overwrite the default config files
+within the apache2 service directory. Sudo permissions are needed for this operation. After these files
+have been overwritten, restart the apache service, and the proxy will be in place. You may want to make
```

Comment:
perhaps adding the backup before the overwrite sentence

Reply:
Fixed. Added Note above.